### PR TITLE
Updated room-invitation section 

### PR DESCRIPTION
**Pull request by shraddhavp**
this PR links to #890  and updates the room invitation section

### CODE REVIEW

File: docusaurus/docs/api/room-invitation.md [hunk 1]
```diff
@@ -1,28 +1,22 @@
 ---
-title: RoomInvitation
+title: Room Invitation
 ---
 
-Accept room invitation
+## Room Invitation
 
-## RoomInvitation
+Room Invitation is a  global class that accepts room invitation.This section gives a description of the methods of the Room Invitation class.
```
Comment (proudofsimin) [inline docusaurus/docs/api/room-invitation.md]:
```suggestion
Room Invitation is a global class that accepts room invitation. This section describes the methods of the Room Invitation class.
```

File: docusaurus/docs/api/room-invitation.md [hunk 2]
```diff
@@ -1,28 +1,22 @@
...
+| topic\(\)    | `Promise` (String)   |
+| date\(\)    | `Promise` (Date)   |
+| age\(\)    | `Promise` (Number)   |
 
 ### roomInvitation.accept\(\) ⇒ `Promise <void>`
 
-Accept Room Invitation
-
-**Kind**: instance method of [`RoomInvitation`](room-invitation.md#RoomInvitation)  
+This method accepts the room invitation.See the following example:
```
Comment (proudofsimin) [inline docusaurus/docs/api/room-invitation.md]:
```suggestion
This method accepts the room invitation. See the following example:
```

File: docusaurus/docs/api/room-invitation.md [hunk 3]
```diff
@@ -41,9 +35,7 @@ bot.on('room-invite', async roomInvitation => {
 
 ### roomInvitation.inviter\(\) ⇒ `Promise <Contact>`
 
-Get the inviter from room invitation
-
-**Kind**: instance method of [`RoomInvitation`](room-invitation.md#RoomInvitation)  
+This method gets the inviter from the room invitation. Check the following example below :
```
Comment (proudofsimin) [inline docusaurus/docs/api/room-invitation.md]:
```suggestion
This method gets the inviter from the room invitation. Check the following example:
```

**Review by proudofsimin — APPROVED** [review]

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]

**Comment by shraddhavp** [comment]
@Soumi7

**Review by Soumi7 — APPROVED** [review]
Lgtm

**Comment by shraddhavp** [comment]
@huan

**Review by huan — APPROVED** [review]
LGTM

Thank you very much for the improvements!

**Pull request merged**